### ISSUE TITLE
[SF][A11y][MessageView]: [The info and warning icons in MessageView is lack of alt text values]

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#7087]
### Describe the bug

The info and warning icons in MessageView component is lack of alt text values

![Image](https://github.com/user-attachments/assets/fe9f97b0-c9b1-4380-83fa-1c173195cf44)

Expected:
Info and warning icons should have text alternatives

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/user-feedback-messageview--docs

### Expected Behaviour

Info and warning icons should have accessible name

### UI5 Web Components for React Version

2.8.0

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @tandan0813 

these icons are only decorative and therefore shouldn't be picked up by screen readers. The linked PR will fix this and add a status announcement for screen readers to each `MessageItem`.
